From c778848b09df073b9f84f22622ea7a632642c4a5 Mon Sep 17 00:00:00 2001
From: David Seifert <soap@gentoo.org>
Date: Tue, 30 Jan 2018 11:13:53 +0100
Subject: [PATCH] Fix pkg-config

* All uses of Catch involve #include'ing it without
  a prefixed folder, such as `#include "catch.hpp"`.
  This requires fixing the pkg-config file.
---
 catch.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/catch.pc.in b/catch.pc.in
index abd0b66b1..4f48bc7fb 100644
--- a/catch.pc.in
+++ b/catch.pc.in
@@ -3,4 +3,4 @@ includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 Name: Catch
 Description: Testing library for C++
 Version: @Catch2_VERSION@
-Cflags: -I${includedir}
+Cflags: -I${includedir}/catch
